src/main/java/co/cask/spanner/common/SpannerConfig.java

```diff
@@ -51,15 +51,13 @@
   @Macro
   public String database;
 
-  @Name("serviceFilePath")
   @Description("Path on the local file system to the service account key used for access to Spanner. " +
     "Does not need to be specified when running on a Dataproc cluster. " +
     "When running on other clusters, the file must be present on every node in the cluster.")
   @Macro
   @Nullable
   public String serviceAccountFilePath;
```

Comment:
should rename the variable to serviceFilePath, otherwise it will be inconsistent with other plugins.

Reply:
@albertshau updated it per your suggestion, please take a look again. thanks